django_ses/__init__.py

```diff
@@ -95,6 +96,7 @@ def open(self):
                 'ses',
                 aws_access_key_id=self._access_key_id,
                 aws_secret_access_key=self._access_key,
+                aws_session_token=self._session_token,
```

Comment:
Should this be added to all places where `boto3` client or connections are used? for example in `ses_email_address.py`, `get_ses_statistics.py`, [`dashboard`](https://github.com/django-ses/django-ses/blob/e64606125e7a56900d44f1e4169a991ec4afbbe8/django_ses/views.py#L98)

Reply:
Yes, it should be added, soon i will make changes to all boto3 client initialization and to README.